src/components/TodoForm.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import {Form ,Field} from "react-final-form";
+import {TODO} from "../types";
+import {useAddTodoMutation} from "../redux/apiSlice";
+
+export const TodoForm = (): JSX.Element => {
+
+
+    const [addTodo] = useAddTodoMutation()
+    const onSubmit = (values: Pick<TODO, 'text'>): void => {
```

Comment:
ten :void je tam skoro zbytečný, myslím že v tomhle případě si to umí sám typescript odhadnout